<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelExporter</class>
 <widget class="QWidget" name="ModelExporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="ModelExporterLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="sceneListLabel">
         <property name="text">
          <string>Scene</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sceneList"/>
       </item>
       <item>
        <widget class="QLabel" name="objectListLabel">
         <property name="text">
          <string>Object</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="objectList">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportHeaderButton">
         <property name="text">
          <string>Export as Header</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportBinButton">
         <property name="text">
          <string>Export as Bin</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QOpenGLWidget" name="modelViewGL">
     <property name="baseSize">
      <size>
       <width>320</width>
       <height>240</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
